packages/block-library/src/social-link/README.md

```diff
@@ -0,0 +1,21 @@
+# Social Icon block
+
+The Social Icon is a foundational block that displays various icons linking to different social profiles or sites. Each social service is registered as a variation of this block.
+
+
+## Adding new social icons
+
+The core maintains certain standards for social icons. To add a new one to the existing variations, it must be a well-established, popular, and mainstream service.
+
+When considering establishment and popularity, several factors should be evaluated, such as:
```

Comment:
To evaluate if a social service should to be added, contributors will consider the following factors.